robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowAccessibilityEvent.java

```diff
@@ -21,6 +24,9 @@
   private static final Map<StrictEqualityEventWrapper, StackTraceElement[]> obtainedInstances =
       new HashMap<>();
 
+  private static final SparseArray<StrictEqualityEventWrapper> orderedInstances = new SparseArray<>();
```

Comment:
Why do we need a second copy of obtainedInstances? It seems to clean up creation from a parcel, but elsewhere you iterate over the array.